### ISSUE TITLE
Releasing training script

### ISSUE THREAD

**Issue by xabirizar9** [hubertsiuzdak/snac#31]
Hi, 

any plans for releasing the training script? I have been able to closely replicate the results in SNAC by following the training procedure in DAC, but I never quite get to the same results, especially for STFT loss.

Was there any extra step that was followed for training SNAC that is not outlined in DAC or mentioned in the paper? Or is the training exactly the same, including:

- Data processing transformations
- The discriminators used during training (same MPD and MRD setup as in DAC)
- Weighting of the different losses other than what's mentioned in the paper.

Would really appreciate an insight into this @hubertsiuzdak ! Good stuff!

**Comment by yukiarimo** [comment]
+1

**Comment by feibuguocanghai** [comment]
+1

**Comment by yukiarimo** [comment]
Updates?

**Comment by feibuguocanghai** [comment]
The latest commit is 2 years ago, it is unlikely to receive an update.
